package.json

```diff
@@ -1,44 +1,82 @@
 {
-  "name": "transcendence",
-  "version": "0.0.1",
-  "description": "42's Common core end project",
-  "main": "src/index.js",
-  "directories": {
-    "doc": "doc"
-  },
-  "scripts": {
-    "build": "webpack",
-    "build:dev": "webpack --watch"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/DoomDuck/transcendence.git"
-  },
-  "keywords": [
-    "42",
-    "transcendance",
-    "nestjs",
-    "pong"
-  ],
-  "contributors": [
-    "dburgun",
-    "efarin",
-    "hgallien",
-    "lzins"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/DoomDuck/transcendence/issues"
-  },
-  "homepage": "https://github.com/DoomDuck/transcendence#readme",
-  "devDependencies": {
-    "css-loader": "^6.7.1",
-    "mini-css-extract-plugin": "^2.6.0",
-    "svelte": "^3.48.0",
-    "svelte-loader": "^3.1.2",
-    "svelte-preprocess": "^4.10.6",
-    "typescript": "^4.7.2",
-    "webpack": "^5.72.1",
-    "webpack-cli": "^4.9.2"
-  }
+	"name": "transcendence",
+	"version": "0.0.1",
+	"description": "42's Common core end project",
+	"directories": {
+		"doc": "doc"
+	},
+	"scripts": {
+		"front_build": "webpack",
+		"front_build:dev": "webpack --watch",
+		"nest_build": "nest build --path tsconfig.back.json",
+		"nest_start": "nest start --path tsconfig.back.json",
+		"nest_start:dev": "nest start --path tsconfig.back.json --watch",
+		"nest_start:debug": "nest start --path tsconfig.back.json --debug --watch",
+		"nest_start:prod": "node build/back/main.js",
+		"_pongserv_build": "tsc -p src/pong/server/tsconfig.json",
+		"_pongserv_build:dev": "tsc -p src/pong/server/tsconfig.json -w",
+		"_pongserv_start": "node build/pong-temporary-server/server/server.js"
```

Comment:
I think those scripts are not needed anymore

Reply:
noted, will be fixed